velox/functions/prestosql/tests/ArrayContainsTest.cpp

```diff
@@ -316,4 +317,61 @@ TEST_F(ArrayContainsTest, dictionaryEncodingElements) {
   testContainsConstantKey(arrayVector, {3, 4, 5}, {true, true});
 }
 
+TEST_F(ArrayContainsTest, arrayCheckNulls) {
+  const auto baseVector = makeArrayVectorFromJson<int32_t>({
```

Comment:
Would you add makeNestedArrayVectorFromJson helper function?

```
makeNestedArrayVectorFromJson<int64_t>({
   "[[1, 1], [2, 2]]",
   ...
});
```

Reply:
@mbasmanova Be happy to. But I remember you used to create an issue https://github.com/facebookincubator/velox/issues/6896 and someone from IBM took over this task.

Reply:
I see that it has been 3 weeks ago and no progress. I assume they changed their mind. Let's double check and take over. This is necessary to make tests readable.